backend/baton/src/test/java/touch/baton/document/runnerpost/read/RunnerPostUpdateSupporterCancelationApiTest.java

```diff
@@ -0,0 +1,98 @@
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.header;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static touch.baton.fixture.vo.CompanyFixture.company;
+import static touch.baton.fixture.vo.GithubUrlFixture.githubUrl;
+import static touch.baton.fixture.vo.ImageUrlFixture.imageUrl;
+import static touch.baton.fixture.vo.MemberNameFixture.memberName;
+import static touch.baton.fixture.vo.OauthIdFixture.oauthId;
+import static touch.baton.fixture.vo.SocialIdFixture.socialId;
+
+@WebMvcTest(RunnerPostController.class)
+public class RunnerPostUpdateSupporterCancelationApiTest extends RestdocsConfig {
+
+    @MockBean
+    private RunnerPostService runnerPostService;
+
+    @BeforeEach
+    void setUp() {
+        final RunnerPostController runnerPostController = new RunnerPostController(runnerPostService);
+        restdocsSetUp(runnerPostController);
+    }
+
+    @DisplayName("러너 게시글에 리뷰 제안한 서포터가 리뷰 제안 철회 API")
+    @Test
+    void updateSupporterCancelRunnerPost() throws Exception {
+        // given
+        final String socialId = "ditooSocialId";
+        final Member member = MemberFixture.create(
+                memberName("디투"),
+                socialId(socialId),
+                oauthId("abcd"),
+                githubUrl("naver.com"),
+                company("우아한테크코스"),
+                imageUrl("profile.jpg")
+        );
+        final Supporter supporter = SupporterFixture.create(member);
+        final String token = getAccessTokenBySocialId(socialId);
+
+        final Runner runner = RunnerFixture.createRunner(MemberFixture.createHyena());
+        final RunnerPost runnerPost = RunnerPostFixture.create(runner, supporter, new Deadline(LocalDateTime.now().plusHours(10)));
+        final RunnerPost spyRunnerPost = spy(runnerPost);
+
+        // when
+        when(spyRunnerPost.getId()).thenReturn(1L);
+        when(oauthSupporterRepository.joinByMemberSocialId(any()))
+                .thenReturn(Optional.ofNullable(supporter));
+        runnerPostService.deleteSupporterRunnerPost(any(), eq(1L));
+
+        // then
+        mockMvc.perform(patch("/api/v1/posts/runner/{runnerPostId}/cancelation", 1L)
+                        .header(AUTHORIZATION, "Bearer " + token))
+                .andExpect(status().isNoContent())
+                .andExpect(header().string("Location", "/api/v1/posts/runner/1"))
+                .andDo(restDocs.document(
+                        requestHeaders(
+                                headerWithName(AUTHORIZATION).description("Bearer JWT")
+                        ),
+                        pathParameters(
+                                parameterWithName("runnerPostId").description("러너 게시글 식별자값")
+                        ),
+                        responseHeaders(headerWithName(LOCATION).description("redirect uri"))
```

Comment:
```suggestion
                        responseHeaders(
                                headerWithName(LOCATION).description("redirect uri")
                         )
```

별거는 아니지만 위에 다 띄어져 있어서 슬쩍 커멘트 남겨봐요